pkg/services/alerting/engine.go

```diff
@@ -115,7 +122,14 @@ func (e *Engine) processJob(grafanaCtx context.Context, job *Job) error {
 
 		e.evalHandler.Eval(evalContext)
 		e.resultHandler.Handle(evalContext)
+		span.LogFields(
+			tlog.Int64("alertId", evalContext.Rule.Id),
+			tlog.Int64("dashboardId", evalContext.Rule.DashboardId),
+			tlog.Bool("firing", evalContext.Firing),
+		)
```

Comment:
if each alert rule invocation gets its own span, then you should set these as tags